src/sql/workbench/contrib/dashboard/browser/pages/dashboardPageContribution.ts

```diff
@@ -57,6 +57,10 @@ export function generateDashboardWidgetSchema(type?: 'database' | 'server', exte
 				properties: schemas,
 				minItems: 1,
 				maxItems: 1
+			},
+			showHeader: {
```

Comment:
nit: I'd prefer this be hideHeader - usually you want the options you specify to be something you're enabling.